src/rnpkeys/rnpkeys.c

```diff
@@ -355,7 +355,10 @@ parse_option(rnp_cfg_t *cfg, int *cmd, const char *s)
 
     if (!compiled) {
         compiled = 1;
-        (void) regcomp(&opt, "([^=]{1,128})(=(.*))?", REG_EXTENDED);
+        if (regcomp(&opt, "([^=]{1,128})(=(.*))?", REG_EXTENDED) != 0) {
+            fprintf(stderr, "Can't compile regex\n");
```

Comment:
@catap That's good question and I've no clue. Let's not free in error case as it may cause crash. 
In fact man says nothing about memory allocation. ``regfree()`` crashes (on my machine) if called on unallocated ``regex_t`` which is quite weird.